src/node_file.cc

```diff
@@ -769,14 +769,14 @@ void Access(const FunctionCallbackInfo<Value>& args) {
   BufferValue path(env->isolate(), args[0]);
   CHECK_NE(*path, nullptr);
 
-  FSReqBase* req_wrap = GetReqWrap(env, args[2]);
-  if (req_wrap != nullptr) {  // access(path, mode, req)
-    AsyncCall(env, req_wrap, args, "access", UTF8, AfterNoArgs,
+  FSReqBase* req_wrap_async = GetReqWrap(env, args[2]);
```

Comment:
Just realized it would be even easier to understand if `FSReqBase` is `FSReqWrapAsync`..LGTM with or without that though.

Reply:
Sorry about the late reply on this. I don't think that is possible with as GetReqWrap can also return FSReqPromise.